package.json

```diff
@@ -25,6 +25,9 @@
   "homepage": "https://github.com/exoplay/exobot-adapter-discord#readme",
   "dependencies": {
     "@r/build": "git://github.com/ajacksified/node-build#679c3f3d47c0916237d55128ece4cc817b489ef8",
+    "babel-plugin-syntax-decorators": "^6.8.0",
```

Comment:
The babel-plugin dependencies should be under "devDependencies" instead of "dependencies"